docker/publish.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+cd $ROOT_DIR/docker
+
+# We should only publish images that are made from official and approved releases
+# and thus ignore all the release candidates that are just tagged during the release
+# process
+if [[ ${TRAVIS_TAG} == *"candidate"* || ${TRAVIS_TAG} == *"rc"* ]]; then
```

Comment:
where are we setting `TRAVIS_TAG`, was it set by travis when we were using it?

Reply:
Yes, this was working on Travis. It hasn't been ported to Jenkins yet. I did manually published the last release.

Reply:
I mean, the script still works. It's just that we need to chose when to trigger it. Eg: right now we need to cut the release, get it approved and then publish it, so it's a different workflow from the Travis based release where everything was done in 1 shot.